src/bus/index.spec.js

```diff
@@ -298,4 +298,67 @@ describe('Bus', function() {
       }
     );
   });
+
+  describe('getWebSocketConnection', function() {
+    let bus;
+    let expectedOrganization;
+
+    beforeEach(function() {
+      expectedOrganization = fixture.build('organization');
+
+      bus = new Bus(baseSdk, baseRequest);
+      bus._webSockets[expectedOrganization.id] = new WebSocketConnection(
+        new WebSocket(
+          `wss://${faker.internet.domainName()}/organizations/${
+            expectedOrganization.id
+          }/stream`
+        ),
+        expectedOrganization.id
+      );
+    });
+
+    context('when an organization id is passed in', function() {
+      context('when a websocket connection exists', function() {
+        let expectedReturnedSocket;
+
+        beforeEach(function() {
+          expectedReturnedSocket = bus.getWebSocketConnection(
+            expectedOrganization.id
+          );
+        });
+
+        it('returns the socket', function() {
+          expect(expectedReturnedSocket).to.deep.equal(
```

Comment:
What do you think about changing the way this is named? It seems like `bus._webSockets[expectedOrganization.id]` should be the expected socket and the return should just be the web socket connection? Like, the return isn't what is expected and now I'm rambling

Reply:
👍